1-exercises/E-for-loop/exercise2.js

```diff
@@ -26,6 +26,13 @@ const AGES = [
     49
 ];
 
+for (let i = 0; i < WRITERS.length; i++) {    
+    for (let j = 0; j < AGES.length; j++) {
```

Comment:
You can remove line 30 and instead of [j] in line  31 use [i] and remove line 32,  you miss years old in the end.